app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ProjectsController < ApplicationController
+  def index
+    @projects = Project.all
+  end
+
+  def new
+    @project = Project.new
+  end
+
+  def create
+    @project = Project.new(project_params)
+    if @project.save
+      flash[:notice] = 'Sucessfully Added'
+      redirect_to projects_path
+    else
+      flash[:notice] = @project.errors.full_messages.join('. ')
+      render :new
+    end
+  end
+
+  def edit
+    @project = Project.find(params[:id])
+  end
+
+  def update
+    @project = Project.find(params[:id])
+    if @project.update_attributes(project_params)
+      flash[:notice] = 'Successfully Updated'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.